doc/Motr_Terminology_Guide.rst

```diff
@@ -0,0 +1,172 @@
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Configuration database                                  | Configuration database is a central repository of cluster configuration.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
++---------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Configuration cache                                     | Configuration cache is configuration data being stored in node’s memory.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
- This glossary has two ‘Configuration cache’ entries.  There should be only one.
- The glossary should be sorted to avoid this kind of problems and simplify the search.